milatools/cli/commands.py

```diff
@@ -418,7 +421,16 @@ def init():
         if yn(
             "Your public key does not appear be registered on the cluster. Register it?"
         ):
-            here.run("ssh-copy-id", "mila")
+            # NOTE: If we're on a Windows machine, we do something different here:
+            if sys.platform == "win32":
+                command = (
+                    "powershell.exe type $env:USERPROFILE\\.ssh\\id_rsa.pub | ssh mila "
+                    '"cat >> ~/.ssh/authorized_keys"'
+                )
+                print(T.bold_green("(local) $ " + command))
+                subprocess.getoutput(command)
```

Comment:
Why not `here.run(command)`?

Reply:
I recall it not working for some reason. I'll try again on Windows and see if it works.

Reply:
Fixed in 78fde43c3a89338f60ef0a2968e4b558132f2817